.sqlfluff 2

```diff
@@ -0,0 +1,32 @@
+[sqlfluff]
```

Comment:
Why is this a new file in your branch? Maybe your local main branch that you based this branch on is several commits behind?